sdk/stdlib-sys/src/intrinsics/debug.rs

```diff
@@ -0,0 +1,14 @@
+#[link(wasm_import_module = "miden:core-import/intrinsics-debug@1.0.0")]
```

Comment:
This interface with the `break` function should be in our core-import WIT file at `tests/rust-apps-wasm/rust-sdk/wit-sdk/miden-core-import.wit`, otherwise wit-component errors on core module import. I'm in the middle of splitting our WIT files and moving them to SDK crates at #505, and I can take care of it there.

Reply:
Ah noted, I'll leave the fix to you then!